src/components/EditorFieldTemplate/index.tsx

```diff
@@ -26,6 +27,14 @@ export default function EditorFieldTemplate(props: FieldTemplateProps) {
   const canAddField = schema.type !== 'object';
   const canAddSection = isSection && !isRoot && !verna.selector;
 
+  const messages = defineMessages({
+    parameters: {
+      defaultMessage: 'Parameters',
+      description: 'Label of the button to display the widget properties form',
+      id: 'components.EditorFieldTemplate.parameters',
+    },
+  });
+
```

Comment:
Declare it outside the component to prevent to recreate this object. We do want to recreate this object at each render.